apps/frontend-v3/lib/vebal/vote/VoteList/SelectedPoolsMenu.tsx

```diff
@@ -13,56 +14,119 @@ interface SelectedPoolsToastProps {
 
 export function SelectedPoolsMenu({ onAddVotesClick, votingPools }: SelectedPoolsToastProps) {
   const [expanded, setExpanded] = useState(false)
+  const [shouldScrollTop, setShouldScrollTop] = useState(false)
   const sortedPools = votingPools.toReversed()
   const visiblePools = expanded
     ? sortedPools.slice(0, MAX_VISIBLE_POOLS_COUNT)
     : sortedPools.slice(0, MIN_VISIBLE_POOLS_COUNT)
 
   const moreCount = sortedPools.length - visiblePools.length
+  const { colorMode } = useColorMode()
+  const bgBase = colorMode === 'dark' ? '#000' : '#fff'
 
-  const [_bgColor] = useToken('colors', ['background.level0'])
-  const bgColor = tinycolor(_bgColor).setAlpha(0.4).toRgbString()
+  const handleGoToVote = () => {
```

Comment:
You don't really need a useEffect here. Something like the following should be ok:

```
  const handleGoToVote = () => {
    setExpanded(false)
    setTimeout(() => {
      window.scrollTo({ top: 0, behavior: 'smooth' })
    })

    onAddVotesClick?.()
  }
```